packages/babel-helper-create-class-features-plugin/src/fields.ts

```diff
@@ -839,8 +865,8 @@ export function buildFieldsInitNodes(
 ) {
   let needsClassRef = false;
   let injectSuperRef: t.Identifier;
-  const staticNodes: t.Statement[] = [];
-  const instanceNodes: t.Statement[] = [];
+  const staticNodes: Array<t.Expression | t.Statement> = [];
+  const instanceNodes: Array<t.Expression | t.Statement> = [];
```

Comment:
This is a bug! Those nodes are inserted in statement positions, so they should always be statements. `buildPrivateInstanceMethodInitSpec` should return `t.expressionStatement(expr)` instead of just `expr`.

Reply:
Seems that typings are not only convenient for writing codes, but also for reviewing codes :)